docs/catalog/web/enable-text-compression.md

```diff
@@ -0,0 +1,45 @@
+---
+version: 1.0
+submitted_by: dubrie
+published_date: TBD
+category: web
+tags: 
+ - text
+ - compression
+ - size:small
+---
+
+# Enable Text Compression
+
+## Description
+
+Web browsers often communicate with Web servers in a human readable format. These can be HTML, JavaScript and/or CSS - files and also REST requests which can return a response in JSON. This communication is human readable and therefore highly redudandant, therefore it can be highly compressed saving a lot of bandwidth.
+
+Web browsers and servers can communicate the compression format dynamically via the "Accept-Encoding"/"Content-Encoding" HTTP Headers. This allows the communication to dynamically switch to compression formats both side support.
+
+
+## Solution
+
+Enable all supported compression formats on the server, this allows clients to use the best available format.
+
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Sofware Carbon Intensity Spec](https://github.com/Green-Software-Foundation/software_carbon_intensity)
```

Comment:
I suggest leaving the link out and perhaps including it more generally in the website (it's going to get redundant to link it on every patter) if we want to keep it i'd suggest using grnsft.org/sci - this will be pointed to the eventual website that will be created for the SCI.

Reply:
Updated here and on #67